dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -156,6 +158,22 @@ public static void registerZenodoDOIForWorkflow(ApiClient zenodoClient, String d
 
         Deposit publishedDeposit = publishDepositOnZenodo(actionsApi, depositionID);
 
+        Object links = publishedDeposit.getLinks();
+        String conceptDoiUrl = (String)((LinkedHashMap)links).get("conceptdoi");
```

Comment:
Can `links` be null? Can `conceptDoiUrl` be null?

Reply:
I don't think links can be null and I don't think the conceptDoiUrl can be null. A conceptDoiUrl is should always exist for a published DOI